source/1_engine/rex_engine/src/filesystem/vfs.cpp

```diff
@@ -311,6 +286,42 @@ namespace rex
           });
     }
 
+    void init(rsl::string_view root)
+    {
+      if(rsl::optional<rsl::string_view> arg = cmdline::get_argument("Root"))
+      {
+        root = arg.value();
+      }
+
+      if(root.empty())
+      {
+        WIN_CALL(GetCurrentDirectoryA(g_root.max_size(), g_root.data()));
```

Comment:
This should not be within the VFS directly.
This should be shielded behind a platform layer

Reply:
not part of this PR I'd say

Reply:
No, but still has to be fixed IMO, this is a platform call that should not be here.
Although this is added code on this branch, maybe it came from a merge into this branch.